.github/workflows/reusable_workflow.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - name: Install arcaflow-docsgen
         env:
-          DOCSGEN_VER: "0.3.0"
+          DOCSGEN_VER: "0.3.1"
```

Comment:
This should really be an input variable -- it shouldn't be defined here, such that we have to keep updating it.

(However, perhaps this is out of scope for this PR....)

Reply:
Let me see if there is a quick fix for this.

Reply:
I can't find a quick solution for this. The reusable workflow is going to run in the context of the calling repo, so `${{ github.ref_name }}` is going to return the version of the calling repo, not the version of the called repo (arcaflow-docsgen). If we change it to an input variable, then we need to change every call to the reusable workflow to pass that variable, and we have to maintain that at each point, which would be a drift maintenance nightmare. There also doesn't seem to be a built-in way to refer to the latest release within a workflow like this.